### DISCUSSION TITLE
Non-unique account numbers?

### DISCUSSION THREAD

**Discussion by bdeshi** [firefly-iii/firefly-iii#6598] [General]
hi.
i'm tracking some mobile banking service accounts in firefly. these services are tied to my phone number, so the account number is essentially the same for these accounts: my phone number.
firefly doesn't allow same account numbers on multiple [asset] accounts, but i think my use case is a valid one. any plans to relax the account number uniqueness? any thoughts?
cc @JC5

**Comment by JC5** [comment]
That's a no from me, sorry about that.
